src/BloomBrowserUI/bookEdit/js/readerToolsModel.ts

```diff
@@ -1002,6 +1002,21 @@ class ReaderToolsModel {
       return libsynphony.selectGPCWordsFromCache(desiredGPCs, knownGPCs, restrictToKnownGPCs, allowUpperCase, syllableLengths, selectedGroups, partsOfSpeech);
   }
 
+  static selectWordsFromAllowedLists(stageNumber: number): string[] {
+
+    var stages: ReaderStage[] = model.getSynphony().getStages(stageNumber);
+
+    var words: string[] = [];
+    for (var i=0; i < stages.length; i++) {
+      if (stages[i].allowedWords)
+        words = words.concat(stages[i].allowedWords);
+    }
+
+    words.sort();
+
+    return _.uniq(_.compact(words), true);
```

Comment:
In that case I'd say we should either do a case-insensitive sort, or go ahead and convert everything to LC before we sort and uniq. The latter gives a more accurate idea of the effect, but might be confusing for words like Susan that are only valid capitalized. So I lean towards the case-insensitive search. @hatton, any thoughts?